grafana_dashboards/components/panels.py

```diff
@@ -29,21 +29,28 @@ def __init__(self, data, registry):
 
 
 class PanelsItemBase(JsonGenerator):
-    pass
+    _default_span = None
 
+    def copy_basic_fields(self, source, dest):
+        for field in {'description', 'transparent', 'repeat'}:
```

Comment:
There is already support for smth similar in `JsonGenerator` through `_copy_fields` attribute. It's already defined in e.g. `Graph` class. I think it should be pretty simple to define commons fields in `PanelsItemBase` and just extend it in all subclasses.